src/cozyUtils.spec.js

```diff
@@ -304,4 +305,228 @@ describe('CozyUtils', () => {
       })
     })
   })
+
+  describe('synchronizeShortcuts', () => {
+    it('should not call deleteFilePermanently if no app to delete', async () => {
+      const foundShortcuts = []
+      const computedShortcuts = {
+        schoolShortcuts: [],
+        favShortcuts: []
+      }
+      const deleteFilePermanentlySpy = jest.fn()
+      cozyUtils.client.collection = jest.fn(() => ({
+        deleteFilePermanently: deleteFilePermanentlySpy
+      }))
+
+      await cozyUtils.synchronizeShortcuts(foundShortcuts, computedShortcuts)
+      // expect(deleteFilePermanentlySpy).toHaveBeenCalledWith([
+      //   appToDelete
+      // ])
+      expect(deleteFilePermanentlySpy).not.toHaveBeenCalled()
+    })
+    it('should delete one app with deleteFilePermanently', async () => {
+      const foundShortcuts = [
+        {
+          _id: 'SOME_ID',
+          type: 'file',
+          metadata: {
+            fileIdAttributes: 'SOME_VALUE'
+          }
+        }
+      ]
+      const computedShortcuts = {
+        schoolShortcuts: [
+          {
+            vendorRef: 'SOME_OTHER_VALUE'
+          }
+        ],
+        favShortcuts: []
+      }
+
+      const deleteFilePermanentlySpy = jest.fn()
+      cozyUtils.client.collection = jest.fn(() => ({
+        deleteFilePermanently: deleteFilePermanentlySpy
+      }))
+
+      await cozyUtils.synchronizeShortcuts(foundShortcuts, computedShortcuts)
+      expect(cozyUtils.client.collection).toHaveBeenCalledWith(DOCTYPE_FILES)
```

Comment:
This one is not necessarily. It doesn't test anything. It's an internal stuff to check that collection has been called. You should remove it because the day we change this internal stuff for cozy-client, your test will failed 

What is really important is the next test aka  `      expect(deleteFilePermanentlySpy).toHaveBeenCalledWith('SOME_ID')`